res/sections/35-Routing.tex

```diff
@@ -190,16 +190,16 @@ \subsubsection{AODV (Ad hoc On-Demand Distance Vector Routing)}
 \begin{itemize}
 \item discovers routes as and where necessary
 \item routes are maintained just as long as necessary
-\item sequence number gets increased every time the node notices changes in the neighbourhood topology
+\item sequence number increases every time the node notices changes in the neighbourhood topology
 \end{itemize}
 
 \paragraph*{How a package is sent} AODV use the following procedure to send a
 packet:
+\begin{multicols}{2}
```

Comment:
Are you sure about this? Because this is the final output:
![image](https://user-images.githubusercontent.com/5928183/35859860-08dcf034-0b42-11e8-9073-30e4b2471bec.png)

I don't know if it's really readable :confused:

Reply:
Yes it is.